### ISSUE TITLE
[3.0.2] darktable hung on opening file

### ISSUE THREAD

**Issue by Germano0** [darktable-org/darktable#5568]
In Fedora KDE, I selected a CR2 file, "open with" -> darktable. Then darktable started and hung

**Platform:**
 - Darktable Version: 3.0.2
 - OS: Fedora 32
 - OpenCL disabled
 - VGA Intel Corporation 2nd Generation Core Processor Family Integrated Graphics Controller (rev 09)

**GDB backtrace**

```
futex_wait_cancelable (private=0, expected=0, futex_word=0x563d428dac80 <darktable+3136>) at ../sysdeps/nptl/futex-internal.h:183
183       int err = lll_futex_timed_wait (futex_word, expected, NULL, private);
(gdb) set height 0
(gdb) set print elements 0
(gdb) set print frame-arguments all
(gdb) thread apply all backtrace

Thread 19 (Thread 0x7ff563478700 (LWP 115546)):
#0  syscall () at ../sysdeps/unix/sysv/linux/x86_64/syscall.S:38
#1  0x00007ff5bda496bc in g_mutex_lock_slowpath (mutex=mutex@entry=0x7ff563469190) at ../glib/gthread-posix.c:1458
#2  0x00007ff5bda4a1c6 in g_mutex_lock (mutex=mutex@entry=0x7ff563469190) at ../glib/gthread-posix.c:1482
#3  0x00007ff5be0c7277 in dt_lua_gtk_wrap_callback (data=0x7ff563469180) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:642
#4  0x00007ff5bd9fca85 in g_main_context_invoke_full (context=0x563d4319eb80, priority=0, function=0x7ff5be0c7240 <dt_lua_gtk_wrap_callback>, data=0x7ff563469180, notify=0x0) at ../glib/gmain.c:5971
#5  0x00007ff5bd9fcb04 in g_main_context_invoke (context=<optimized out>, function=<optimized out>, data=<optimized out>) at ../glib/gmain.c:5916
#6  0x00007ff5be0c7373 in gtk_wrap (L=0x7ff55c003398) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:669
#7  0x00007ff5bb4dd21f in luaD_precall (L=L@entry=0x7ff55c003398, func=0x7ff55c003a30, nresults=0) at ldo.c:434
#8  0x00007ff5bb4dd5af in luaD_call (nResults=1543517080, func=<optimized out>, L=0x7ff55c003398) at ldo.c:498
#9  luaD_callnoyield (L=L@entry=0x7ff55c003398, func=<optimized out>, nResults=nResults@entry=0) at ldo.c:509
#10 0x00007ff5bb4d3500 in lua_callk (L=L@entry=0x7ff55c003398, nargs=nargs@entry=1, nresults=nresults@entry=0, ctx=ctx@entry=0, k=k@entry=0x0) at lapi.c:925
#11 0x00007ff561f62083 in lua_update_metadata (L=0x7ff55c003398) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/libs/metadata_view.c:808
#12 0x00007ff5bb4dd21f in luaD_precall (L=L@entry=0x7ff55c003398, func=0x7ff55c0034c0, nresults=0) at ldo.c:434
#13 0x00007ff5bb4dd5af in luaD_call (nResults=<optimized out>, func=<optimized out>, L=0x7ff55c003398) at ldo.c:498
#14 luaD_callnoyield (L=0x7ff55c003398, func=<optimized out>, nResults=<optimized out>) at ldo.c:509
#15 0x00007ff5bb4dc937 in luaD_rawrunprotected (L=L@entry=0x7ff55c003398, f=f@entry=0x7ff5bb4d0e40 <f_call>, ud=ud@entry=0x7ff563469440) at ldo.c:142
#16 0x00007ff5bb4dd98f in luaD_pcall (L=L@entry=0x7ff55c003398, func=func@entry=0x7ff5bb4d0e40 <f_call>, u=u@entry=0x7ff563469440, old_top=80, ef=<optimized out>) at ldo.c:729
#17 0x00007ff5bb4d35ce in lua_pcallk (L=L@entry=0x7ff55c003398, nargs=<optimized out>, nresults=nresults@entry=0, errfunc=errfunc@entry=4, ctx=ctx@entry=0, k=k@entry=0x0) at lapi.c:969
#18 0x00007ff5be0c7d9e in run_async_thread_main (data=<optimized out>, user_data=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:139
#19 0x00007ff5bda26114 in g_thread_pool_thread_proxy (data=<optimized out>) at ../glib/gthreadpool.c:354
#20 0x00007ff5bda25812 in g_thread_proxy (data=0x563d44b0e5e0) at ../glib/gthread.c:807
#21 0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#22 0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 18 (Thread 0x7ff562c77700 (LWP 115545)):
#0  syscall () at ../sysdeps/unix/sysv/linux/x86_64/syscall.S:38
#1  0x00007ff5bda4a303 in g_cond_wait (cond=cond@entry=0x7ff562c68180, mutex=mutex@entry=0x7ff562c68190) at ../glib/gthread-posix.c:1540
#2  0x00007ff5be0c737e in gtk_wrap (L=0x7ff55c002a18) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:670
#3  0x00007ff5bb4dd21f in luaD_precall (L=L@entry=0x7ff55c002a18, func=0x7ff55c002bb0, nresults=0) at ldo.c:434
#4  0x00007ff5bb4dd5af in luaD_call (nResults=1543514648, func=<optimized out>, L=0x7ff55c002a18) at ldo.c:498
#5  luaD_callnoyield (L=L@entry=0x7ff55c002a18, func=<optimized out>, nResults=nResults@entry=0) at ldo.c:509
#6  0x00007ff5bb4d3500 in lua_callk (L=L@entry=0x7ff55c002a18, nargs=nargs@entry=1, nresults=nresults@entry=0, ctx=ctx@entry=0, k=k@entry=0x0) at lapi.c:925
#7  0x00007ff561f62083 in lua_update_metadata (L=0x7ff55c002a18) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/libs/metadata_view.c:808
#8  0x00007ff5bb4dd21f in luaD_precall (L=L@entry=0x7ff55c002a18, func=0x7ff55c002b40, nresults=0) at ldo.c:434
#9  0x00007ff5bb4dd5af in luaD_call (nResults=<optimized out>, func=<optimized out>, L=0x7ff55c002a18) at ldo.c:498
#10 luaD_callnoyield (L=0x7ff55c002a18, func=<optimized out>, nResults=<optimized out>) at ldo.c:509
#11 0x00007ff5bb4dc937 in luaD_rawrunprotected (L=L@entry=0x7ff55c002a18, f=f@entry=0x7ff5bb4d0e40 <f_call>, ud=ud@entry=0x7ff562c68440) at ldo.c:142
#12 0x00007ff5bb4dd98f in luaD_pcall (L=L@entry=0x7ff55c002a18, func=func@entry=0x7ff5bb4d0e40 <f_call>, u=u@entry=0x7ff562c68440, old_top=80, ef=<optimized out>) at ldo.c:729
#13 0x00007ff5bb4d35ce in lua_pcallk (L=L@entry=0x7ff55c002a18, nargs=<optimized out>, nresults=nresults@entry=0, errfunc=errfunc@entry=4, ctx=ctx@entry=0, k=k@entry=0x0) at lapi.c:969
#14 0x00007ff5be0c7d9e in run_async_thread_main (data=<optimized out>, user_data=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:139
#15 0x00007ff5bda26114 in g_thread_pool_thread_proxy (data=<optimized out>) at ../glib/gthreadpool.c:354
#16 0x00007ff5bda25812 in g_thread_proxy (data=0x563d44b0e580) at ../glib/gthread.c:807
#17 0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#18 0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 17 (Thread 0x7ff56447a700 (LWP 115544)):
#0  syscall () at ../sysdeps/unix/sysv/linux/x86_64/syscall.S:38
#1  0x00007ff5bda4a303 in g_cond_wait (cond=cond@entry=0x7ff56446b180, mutex=mutex@entry=0x7ff56446b190) at ../glib/gthread-posix.c:1540
#2  0x00007ff5be0c737e in gtk_wrap (L=0x7ff55c0020b8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:670
#3  0x00007ff5bb4dd21f in luaD_precall (L=L@entry=0x7ff55c0020b8, func=0x7ff55c002250, nresults=0) at ldo.c:434
#4  0x00007ff5bb4dd5af in luaD_call (nResults=1543512248, func=<optimized out>, L=0x7ff55c0020b8) at ldo.c:498
#5  luaD_callnoyield (L=L@entry=0x7ff55c0020b8, func=<optimized out>, nResults=nResults@entry=0) at ldo.c:509
#6  0x00007ff5bb4d3500 in lua_callk (L=L@entry=0x7ff55c0020b8, nargs=nargs@entry=1, nresults=nresults@entry=0, ctx=ctx@entry=0, k=k@entry=0x0) at lapi.c:925
#7  0x00007ff561f62083 in lua_update_metadata (L=0x7ff55c0020b8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/libs/metadata_view.c:808
#8  0x00007ff5bb4dd21f in luaD_precall (L=L@entry=0x7ff55c0020b8, func=0x7ff55c0021e0, nresults=0) at ldo.c:434
#9  0x00007ff5bb4dd5af in luaD_call (nResults=<optimized out>, func=<optimized out>, L=0x7ff55c0020b8) at ldo.c:498
#10 luaD_callnoyield (L=0x7ff55c0020b8, func=<optimized out>, nResults=<optimized out>) at ldo.c:509
#11 0x00007ff5bb4dc937 in luaD_rawrunprotected (L=L@entry=0x7ff55c0020b8, f=f@entry=0x7ff5bb4d0e40 <f_call>, ud=ud@entry=0x7ff56446b440) at ldo.c:142
#12 0x00007ff5bb4dd98f in luaD_pcall (L=L@entry=0x7ff55c0020b8, func=func@entry=0x7ff5bb4d0e40 <f_call>, u=u@entry=0x7ff56446b440, old_top=80, ef=<optimized out>) at ldo.c:729
#13 0x00007ff5bb4d35ce in lua_pcallk (L=L@entry=0x7ff55c0020b8, nargs=<optimized out>, nresults=nresults@entry=0, errfunc=errfunc@entry=4, ctx=ctx@entry=0, k=k@entry=0x0) at lapi.c:969
#14 0x00007ff5be0c7d9e in run_async_thread_main (data=<optimized out>, user_data=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:139
#15 0x00007ff5bda26114 in g_thread_pool_thread_proxy (data=<optimized out>) at ../glib/gthreadpool.c:354
#16 0x00007ff5bda25812 in g_thread_proxy (data=0x563d44b0e520) at ../glib/gthread.c:807
#17 0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#18 0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 16 (Thread 0x7ff563c79700 (LWP 115543)):
#0  0x00007ff5bdd74b6f in __GI___poll (fds=0x563d44abde50, nfds=1, timeout=-1) at ../sysdeps/unix/sysv/linux/poll.c:29
#1  0x00007ff5bd9fbace in g_main_context_poll (priority=<optimized out>, n_fds=1, fds=0x563d44abde50, timeout=<optimized out>, context=0x563d44ac72b0) at ../glib/gmain.c:4346
#2  g_main_context_iterate (context=0x563d44ac72b0, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>) at ../glib/gmain.c:4042
#3  0x00007ff5bd9fbe53 in g_main_loop_run (loop=0x7ff55c000c10) at ../glib/gmain.c:4241
#4  0x00007ff5be0c7478 in lua_thread_main (data=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/call.c:579
#5  0x00007ff5bda25812 in g_thread_proxy (data=0x563d44aa9000) at ../glib/gthread.c:807
#6  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#7  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 15 (Thread 0x7ff588ff9700 (LWP 115531)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbdda4 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work_res (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:529
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 14 (Thread 0x7ff5897fa700 (LWP 115530)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbdda4 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work_res (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:529
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 13 (Thread 0x7ff589ffb700 (LWP 115529)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbdda4 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work_res (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:529
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 12 (Thread 0x7ff58a7fc700 (LWP 115528)):
#0  0x00007ff5bdd46871 in __GI___clock_nanosleep (clock_id=clock_id@entry=0, flags=flags@entry=0, req=req@entry=0x7ff58a7ed4a0, rem=rem@entry=0x7ff58a7ed4a0) at ../sysdeps/unix/sysv/linux/clock_nanosleep.c:48
#1  0x00007ff5bdd4c1c7 in __GI___nanosleep (requested_time=requested_time@entry=0x7ff58a7ed4a0, remaining=remaining@entry=0x7ff58a7ed4a0) at nanosleep.c:27
#2  0x00007ff5bdd4c0fe in __sleep (seconds=0, seconds@entry=2) at ../sysdeps/posix/sleep.c:55
#3  0x00007ff5bdfbd952 in dt_control_worker_kicker (ptr=0x563d4319bd60) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:544
#4  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#5  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 11 (Thread 0x7ff58affd700 (LWP 115527)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 10 (Thread 0x7ff58b7fe700 (LWP 115526)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 9 (Thread 0x7ff58bfff700 (LWP 115525)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 8 (Thread 0x7ff5a8e14700 (LWP 115524)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 7 (Thread 0x7ff5a9615700 (LWP 115523)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 6 (Thread 0x7ff5a9e16700 (LWP 115522)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 5 (Thread 0x7ff5aa617700 (LWP 115521)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 4 (Thread 0x7ff5aae18700 (LWP 115520)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d4319c920) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d4319c8f8, mutex=0x563d4319c8a8) at pthread_cond_wait.c:638
#3  0x00007ff5bdfbf036 in dt_pthread_cond_wait (mutex=0x563d4319c8a8, cond=0x563d4319c8f8) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_control_work (ptr=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/control/jobs.c:572
#5  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#6  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 3 (Thread 0x7ff5ab6bb700 (LWP 115519)):
#0  0x00007ff5bdd74b6f in __GI___poll (fds=0x563d431769d0, nfds=3, timeout=-1) at ../sysdeps/unix/sysv/linux/poll.c:29
#1  0x00007ff5bd9fbace in g_main_context_poll (priority=<optimized out>, n_fds=3, fds=0x563d431769d0, timeout=<optimized out>, context=0x563d431b55b0) at ../glib/gmain.c:4346
#2  g_main_context_iterate (context=0x563d431b55b0, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>) at ../glib/gmain.c:4042
#3  0x00007ff5bd9fbe53 in g_main_loop_run (loop=0x563d43176850) at ../glib/gmain.c:4241
#4  0x00007ff5bce5a6aa in gdbus_shared_thread_func (user_data=0x563d431a7e10) at ../gio/gdbusprivate.c:279
#5  0x00007ff5bda25812 in g_thread_proxy (data=0x563d431af460) at ../glib/gthread.c:807
#6  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#7  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 2 (Thread 0x7ff5ac705700 (LWP 115517)):
#0  0x00007ff5bdd74b6f in __GI___poll (fds=0x563d43169ae0, nfds=1, timeout=-1) at ../sysdeps/unix/sysv/linux/poll.c:29
#1  0x00007ff5bd9fbace in g_main_context_poll (priority=<optimized out>, n_fds=1, fds=0x563d43169ae0, timeout=<optimized out>, context=0x563d431a1870) at ../glib/gmain.c:4346
#2  g_main_context_iterate (context=context@entry=0x563d431a1870, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>) at ../glib/gmain.c:4042
#3  0x00007ff5bd9fbc03 in g_main_context_iteration (context=0x563d431a1870, may_block=may_block@entry=1) at ../glib/gmain.c:4108
#4  0x00007ff5bd9fbc51 in glib_worker_main (data=<optimized out>) at ../glib/gmain.c:5996
#5  0x00007ff5bda25812 in g_thread_proxy (data=0x563d4315c760) at ../glib/gthread.c:807
#6  0x00007ff5bdc65432 in start_thread (arg=<optimized out>) at pthread_create.c:477
#7  0x00007ff5bdd7f9d3 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95

Thread 1 (Thread 0x7ff5b970a140 (LWP 115516)):
#0  futex_wait_cancelable (private=0, expected=0, futex_word=0x563d428dac80 <darktable+3136>) at ../sysdeps/nptl/futex-internal.h:183
#1  __pthread_cond_wait_common (abstime=0x0, clockid=0, mutex=0x563d428dac30 <darktable+3056>, cond=0x563d428dac58 <darktable+3096>) at pthread_cond_wait.c:508
#2  __pthread_cond_wait (cond=0x563d428dac58 <darktable+3096>, mutex=0x563d428dac30 <darktable+3056>) at pthread_cond_wait.c:638
#3  0x00007ff5be0d184f in dt_pthread_cond_wait (mutex=<optimized out>, cond=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/dtpthread.h:327
#4  dt_lua_lock_internal (function=<optimized out>, file=<optimized out>, line=<optimized out>, silent=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/lua/lua.c:146
#5  0x00007ff5bdf5d071 in dt_image_import_internal (film_id=<optimized out>, filename=<optimized out>, override_ignore_jpegs=<optimized out>, lua_locking=1) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/image.c:1332
#6  0x00007ff5bdf18dcf in dt_load_from_string (input=<optimized out>, open_image_in_dr=<optimized out>, single_image=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/darktable.c:257
#7  0x00007ff5bdf1b4b5 in dt_init (argc=<optimized out>, argv=<optimized out>, init_gui=<optimized out>, load_data=<optimized out>, L=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/common/darktable.c:1059
#8  0x0000563d428d70cc in main (argc=<optimized out>, argv=<optimized out>) at /usr/src/debug/darktable-3.0.2-1.fc32.x86_64/src/main.c:92

```

**Comment by TurboGit** [comment]
A possible issue is a Lua script (I see many lua strings in the backtrace). Have you tried without Lua ?

Also at this stage there is zero chance that we will fix 3.0.2 as 3.2 will be out soon. Have you tried with the current master branch ?

**Comment by Germano0** [comment]
Hi Pascal, I cannot reproduce the problem so I cannot do these tries.
Feel free to close the bugreport if the backtrace is not enough to address the problem

**Comment by phrrk** [comment]
There is a current discussion on the darktable-user mailing list;
 [darktable-user] darktable not honouring 'open with darktable'
that suggests that this is a common issue for 3.0.2 and latest git, with windows 10 and several linux distributions

**Comment by c-h-r-i-s-t-i-a-n** [comment]
See also #4469 where I reported a similar problem.

**Comment by ptilopteri** [comment]
I can open nef (raw) files from gimp using darktable if I move or rename ~/.config/darktable/luarc so it is not read.

there is apparently an lua file which is causing difficulty.

**Comment by johnny-bit** [comment]
The fix for blocking lua is in 3.2.1 (as well as updated lua scripts). Closing :)
